src/ir_Mitsubishi.cpp

```diff
@@ -474,6 +489,56 @@ bool IRMitsubishiAC::getiSave10C(void) const {
     return _.iSave10C;
 }
 
+void IRMitsubishiAC::setISee(const bool state) {
+    _.ISee = state;
+}
+
+bool IRMitsubishiAC::getISee(void) const {
+    return _.ISee;
+}
+
+void IRMitsubishiAC::setEcocool(const bool state) {
+    _.Ecocool = state;
+}
+
+bool IRMitsubishiAC::getEcocool(void) const {
+    return _.Ecocool;
+}
+
+void IRMitsubishiAC::setAbsenseDetect(const bool state) {
+    _.AbsenseDetect = state;
+}
+
+bool IRMitsubishiAC::getAbsenseDetect(void) const {
+    return _.AbsenseDetect;
+}
+
+/// Set the requested Direct/Indirect mode. Only works if I-See mode is ON.
+/// @param[in] The requested Direct/Indirect mode..
+void IRMitsubishiAC::setDirectIndirect(const uint8_t mode) {
+    if (_.ISee) {
+        _.DirectIndirect = std::min(mode, kMitsubishiAcDirect);  // bounds check
+    }
+    else {
+        _.DirectIndirect = 0;
+    }
+}
+
+/// Get the LDirect/Indirect mode of the A/C.
+/// @return The native mode setting.
+uint8_t IRMitsubishiAC::getDirectIndirect(void) const { 
+    return _.DirectIndirect;
+}
+
+void IRMitsubishiAC::setNaturalFlow(const bool state) {
+    _.NaturalFlow = state;
+}
+
+bool IRMitsubishiAC::getNaturalFlow(void) const{
+    return _.NaturalFlow;
+}
+
+
```

Comment:
Style-guide-nitpick: Only two (not four) spaces for each level of indentation please.